HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -58,6 +58,13 @@ def customiseFor14833(process):
                 producer.includeME0 = cms.bool(False)
     return process
 
+def customiseFor16670(process):
```

Comment:
it's a typical problem of no easy fix.
There is not fillDescriptions for this tool.

It's an ESProducer, it currently gets a tool to be configured from the plugin factory during ::produce call, the needed parameter is a sub-tool of the tool to be configured.
So, some significant development and some refactoring might be necessary.

@Dr15Jones do we have an example of what to do in this case?